nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -107,9 +109,13 @@ class BtServiceNode : public BT::SyncActionNode
    */
   BT::NodeStatus tick() override
   {
-    on_tick();
```

Comment:
I'm debating this question - do we actually _need_ to change the service node at all? These should be short running computations and the `future_result_` doesn't give you an accepted / declined feedback the way actions do to wait until accepted before continuing (since that ack would actually be the response itself).

Reply:
I agree. The only thing is that it cloud potentially slow down the tick rate if say a server takes some time to respond or the service fails for whatever reason. That being said, I don't think these changes to the BT service nodes would affect the performance in any way, might as well add them to avoid any potential issues in the future?

Reply:
OK, sounds good. I looked over the implementation and looks good to me